walkeeper/src/callmemaybe.rs

```diff
@@ -190,7 +195,7 @@ impl Drop for SubscriptionState {
 }
 
 pub async fn main_loop(conf: SafeKeeperConf, mut rx: UnboundedReceiver<CallmeEvent>) -> Result<()> {
-    let subscriptions: Mutex<HashMap<(ZTenantId, ZTimelineId), SubscriptionState>> =
+    let subscriptions: Mutex<HashMap<(ZTenantId, ZTimelineId, String), SubscriptionState>> =
```

Comment:
NIT: that `String` might be more readable with the name. Also, the joined Id could be used too, as the other components come to the same approach?

Reply:
Done, see last commit. It is weird that the value (SubscriptionState) keeps the entire key too. It should be ok since this shouldn't be a bottleneck here, but doesnt look great anyway